projects/packages/google-fonts-provider/src/class-utils.php

```diff
@@ -44,14 +44,38 @@ public static function font_source_resource_hint( $urls, $relation_type ) {
 	 * @return boolean|void Whether the font family is registered, or void if WP_Webfonts is not available.
 	 */
 	public static function is_font_family_registered( $font_family_name ) {
-		if ( ! function_exists( 'wp_webfonts' ) || ! method_exists( 'WP_Webfonts', 'get_font_slug' ) ) {
-			return;
-		}
+		// New WP Fonts API since Gutenberg 14.9
```

Comment:
```suggestion
		// New WP Fonts API since Gutenberg 14.9
		// Remove conditional once WP 6.2 is the minimum version (must confirm this made it into 6.2)
```